hotsos/defs/scenarios/openstack/nova/bugs/lp1972028.yaml

```diff
@@ -0,0 +1,19 @@
+checks:
+  logs:
+    input:
+      path: 'var/log/nova/nova-compute.log'
+    search:
+      expr: '([\d-]+) ([\d:]+)\.\d{3} .+ libvirt.libvirtError: Node device not found: no node device with matching name \S+'
+      constraints:
+        search-result-age-hours: 168  # 7 days
+conclusions:
+  lp1967956:
+    decision: logs
+    raises:
+      type: LaunchpadBug
+      bug-id: 1972028
+      message: >-
+        This host is affected by an Openstack Nova bug where a race condition
+        when polling libvirt for pci passthrough devices can cause an error to
+        be raised. This is fixed in the latest packages so upgrading is
```

Comment:
Maybe note the specific version instead of "latest"?

Reply:
ill fix the yamllint error tho